Project.toml

```
@@ -72,6 +73,7 @@ CUDA = "5"
 CUDSS = "0.1, 0.2, 0.3"
 ChainRulesCore = "1.22"
 ConcreteStructs = "0.2.3"
+CpuId = "0.3.2"
```

```suggestion
CpuId = "0.3.1"
```
0.3.1 is the latest release.
https://github.com/JuliaRegistries/General/blob/master/C/CpuId/Versions.toml